packages/build/src/core/build.ts

```diff
@@ -124,6 +125,20 @@ const tExecBuild = async function ({
     timers,
     quiet,
   })
+
+  if (featureFlags['build_automatic_runtime'] && framework) {
+    const runtime = supportedRuntimes[framework]
+
+    if (runtime !== undefined) {
+      const dontSkip = childEnv[runtime.skipFlag] !== 'true'
```

Comment:
I find the negative form a bit confusing for a variable name. Can we call this `skip` instead?